### PR TITLE
NMS-14123: Sentinel debian package fail to install

### PR DESCRIPTION
**Pull request by freddy61025**
[jira/NMS-14123](https://issues.opennms.org/browse/NMS-14123)

It is cause by the file remove earlier than fix permission. And, the file list is obtain from debian package. So chown will exit with non 0 status code.

### CODE REVIEW

**Review by cgorantla — APPROVED** [review]
LGTM!

But wait for Ben's approval for merging.

**Review by RangerRick — APPROVED** [review]
sure, this is fine

**Pull request merged**
